src/lucky.cr

```diff
@@ -21,11 +21,6 @@ require "./lucky/paginator/*"
 require "./lucky/paginator/components/*"
 
 module Lucky
-  Habitat.create do
-    setting logger : Dexter::Logger
-  end
-
-  def self.logger
-    settings.logger
-  end
+  Log              = ::Log.for("lucky")
```

Comment:
Technically you can pass `self` but I like the idea of passing a string so we know exactly what the source name is